setup/java.md

```diff
@@ -6,42 +6,54 @@ The [Spring Boot GraphQL Starter](https://github.com/graphql-java-kickstart/grap
 
 ### Requirements
 
-* Java 1.8
+* Java 17
```

Comment:
If this file is identical to the one on PR https://github.com/thinkb4/a-walk-in-graphql/pull/86 then either copy the suggested modifications or rever the changes here and merge this PR after the 86 so you don't have to do it twice 😉